packages/storybook/src/utils/IncludeDocumentation.tsx

```diff
@@ -0,0 +1,25 @@
+import { Markdown } from '@storybook/blocks';
+import { ReactElement } from 'react';
+
+interface DocumentationProps {
+  component: string;
+  url: string;
+  markdown: string;
+}
+
+export const IncludeDocumentation = ({ component, url, markdown }: DocumentationProps): ReactElement => {
```

Comment:
```suggestion
export const SourceCitation = ({ component, url, markdown }: DocumentationProps): ReactElement => {
```

Of iets dergelijks

Reply:
Het voegt ook de Markdown toe, niet alleen het citation-block

Reply:
Zie ook andere comment, Bij de refactor/nieuwe microcontent componenten kan het gesplitst worden en in beide varianten gebruikt worden.